wp-content/themes/mambaone/single-cpt_lessons.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*Template Name: Single cpt Lessons
+*/
+
+get_header();
+
+$args = array(
+    'post_type' => 'cpt_lessons',
+    'orderby' => 'publish_date',
+    'order' => 'DESC',
+    'posts_per_page' => 3,
+);
+
+$loop = new WP_Query($args);
+?>
+<?php
+if ($loop->have_posts()): ?>
+    <ul>
+        <?php
+        while ($loop->have_posts()) : $loop->the_post(); ?>
+            <li>
+                <a href="<?php
+                the_permalink(); ?>"><?php
+                    the_title(); ?></a>
+            </li>
+        <?php
+        endwhile; ?>
+    </ul>
+<?php
+endif; ?>
+
+<?php
+$second_args = array(
```

Comment:
It is good practice to set a limit on the query results.

Reply:
added       'post_status' => 'publish',